pkg/admissioncontroller/webhook/admission/auditpolicy/handler.go

```diff
@@ -227,10 +226,10 @@ func (h *Handler) getOldObject(request admission.Request, oldObj runtime.Object)
 }
 
 func validateAuditPolicySemantics(auditPolicy string) (errCode int32, err error) {
-	return validateAuditPolicySemanticsForKubernetesVersions(auditPolicy, nil)
+	return validateAuditPolicySemanticsForKubernetesVersions(auditPolicy)
 }
 
-func validateAuditPolicySemanticsForKubernetesVersions(auditPolicy string, shootNameToKubernetesVersion map[string]string) (errCode int32, err error) {
+func validateAuditPolicySemanticsForKubernetesVersions(auditPolicy string) (errCode int32, err error) {
```

Comment:
Could you adapt the function name, because it does not consider different Kubernetes versions anymore?